xml/admin_ceph_upgrade.xml

```diff
@@ -1397,6 +1397,146 @@ data:
    </step>
   </procedure>
  </sect1>
+
+ <sect1 xml:id="filestore2bluestore">
+  <title>OSD Migration to &bluestore;</title>
+  <para>
+   OSD &bluestore; is a new back end for the OSD daemons. It is the default
+   option since &productname; &productnumber;. Compared to FileStore, which
```

Comment:
I suspect it does not make sense to specify a particular capability since a specific version of a product (any product) using an updating entity. Surely this is a fixed, specific release and should be explicitly identified?

Reply:
@swiftgist ?

Reply:
Is this question the context?  As in, this is the original SES5 content and the backend was new at the time, but for SES6 this is no longer new.

With respect to referencing Bluestore as the default backend for OSDs, is it better to follow the product, project or not mention it.  I can see the value in mentioning it if part of the goal is to persuade the reader to migrating.